Add the Liveries from the Livery database

IAHM-COL opened the pull request.

Commit:
Removal of the unused svn:mime-type subversion property from all of FGAddon.

This follows from the proposal for deleting the property for all XML files at
http://thread.gmane.org/gmane.games.flightgear.devel/77365 and the discussions
in the thread at http://thread.gmane.org/gmane.games.flightgear.devel/77606 (for
setting up a repository pre-commit hook script for catching this unused
property) where deleting the property for all files was discussed.

This svn:mime-type repository property is only for optimising the back-end
storage size of the db/revs/* files on the SourceForge infrastructure, by
allowing certain text files with a very limited number of extremely long lines
to be treated as a binary rather than a text file, hence suppressing diffs in
the commit.  As no one has deliberately set this and many text files in FGAddon
are accidentally not being treated as text, this change will result in a major
repository clean up which will allow all text files to be treated as text.  The
activation of the pre-commit hook will prevent this problem from ever
reappearing.

This change touches the files in 480 aircraft directories out of 485, however
this does not affect the file contents or aircraft in any way.



git-svn-id: http://svn.code.sf.net/p/flightgear/fgaddon/trunk/Aircraft/747-8i@583 9eb9bc84-db9d-4e05-893b-26dd41ac94a6

IAHM-COL committed:
AUTOMERGE SVN MASTER REV583

IAHM-COL committed:
Merge remote-tracking branch 'upstream/master'

IAHM-COL committed:
Liveries from the Flightgear Livery database added

http://liveries.flightgear.org/aircraft.php?id=127
Authors (Thanks to)
	Emmanuel Savage
	Ahmet Erfkemper

IAHM-COL committed:
Merge remote-tracking branch 'upstream/master'

dogsaysmoo commented:
I'm not going to merge liveries from the database. They all use the file structure from the old fgbr 748 which is not the same as the current version.

IAHM-COL commented:
Thanks for the note too 
Do you say: [b]all?[/b]

I am reverting the livery database additions for both the 757-200 and the 747-8i. 
Thanks for your revsions.

IAHM-COL commented:
are you sure about this one?
I had take a look at every png file and they mount perfectly well over your example textures
Notice that these liveries are under 747-8i in the database: http://liveries.flightgear.org/aircraft.php?id=127
And this are only 3, and are isolated from the 747-400 here:
http://liveries.flightgear.org/aircraft.php?id=1

I looks to me these 3 additions work perfectly

Best,
IH-COL

IAHM-COL commented:
Dear John @dogsaysmoo 

I took the liberty of testing the 3 liveries over the database that I am importing with this pull request.
They all work. 
See images below

@IAHM-COL

![fgfs-screen-065](https://cloud.githubusercontent.com/assets/9952283/7543957/b87e612c-f57d-11e4-95ce-1c5ec6bfa616.png)
![fgfs-screen-066](https://cloud.githubusercontent.com/assets/9952283/7543959/be45fd5e-f57d-11e4-9a48-fa2df1c8d5f4.png)
![fgfs-screen-067](https://cloud.githubusercontent.com/assets/9952283/7543960/c21d9040-f57d-11e4-84d5-d25c3b346162.png)

IAHM-COL committed:
Sound files transferred to monophonic

IAHM-COL commented:
Hi John
@dogsaysmoo 

Last commit changes a few sound files to monophonic so the sound is restored

best
@IAHM-COL

IAHM-COL committed:
Merge remote-tracking branch 'upstream/master'

Commit:
B747-8: Cabin pressurization system, updates, bug fixes

git-svn-id: http://svn.code.sf.net/p/flightgear/fgaddon/trunk/Aircraft/747-8i@721 9eb9bc84-db9d-4e05-893b-26dd41ac94a6

IAHM-COL committed:
Merge remote-tracking branch 'upstream/master'

IAHM-COL committed:
Merge remote-tracking branch 'svn/master'

Commit:
B747-8: New exterior lighting scheme

git-svn-id: http://svn.code.sf.net/p/flightgear/fgaddon/trunk/Aircraft/747-8i@764 9eb9bc84-db9d-4e05-893b-26dd41ac94a6

Commit:
B747-8: New exterior lighting scheme

git-svn-id: http://svn.code.sf.net/p/flightgear/fgaddon/trunk/Aircraft/747-8i@765 9eb9bc84-db9d-4e05-893b-26dd41ac94a6

IAHM-COL committed:
Merge remote-tracking branch 'upstream/master'

IAHM-COL committed:
Merge remote-tracking branch 'svn/master'

Commit:
Manual reversion of r583 and r570 - the removal of the svn:mime-type property from all files.

To avoid the files with the extensions [ac, eff, frag, nas, osgx, svg, txt, vert, xhtml, xml, xsl]
being treated as binary files, as was previously the case, the normal 'svn merge' command was not
used for the reversion.  Instead the svn:mime-type property was manually readded to all the binary
file types using the find command together with 'svn propset'.  Therefore the svn:mime-type
properties of the files may not always match those from before r583.

find -iname "*.wav" -exec svn propset svn:mime-type "audio/x-wav" {} \;
find -iname "*.bmp*" -exec svn propset svn:mime-type "image/x-ms-bmp" {} \;
find -iname "*.gif" -exec svn propset svn:mime-type "image/gif" {} \;
find -iname "*.jpg" -exec svn propset svn:mime-type "image/jpeg" {} \;
find -iname "*.png*" -exec svn propset svn:mime-type "image/png" {} \;
find -iname "*.psd" -exec svn propset svn:mime-type "image/vnd.adobe.photoshop" {} \;
find -iname "*.tif" -exec svn propset svn:mime-type "image/tiff" {} \;
find -iname "*.xcf" -exec svn propset svn:mime-type "image/x-xcf" {} \;
find -iname "*.rgb*" -exec svn propset svn:mime-type "application/octet-stream" {} \;
find -iname "*.au" -exec svn propset svn:mime-type "application/octet-stream" {} \;
find -iname "*.*af" -exec svn propset svn:mime-type "application/octet-stream" {} \;
find -iname "*.3ds" -exec svn propset svn:mime-type "application/octet-stream" {} \;
find -iname "*.blend*" -exec svn propset svn:mime-type "application/octet-stream" {} \;
find -iname "*.dds" -exec svn propset svn:mime-type "application/octet-stream" {} \;
find -iname "*.od[gst]" -exec svn propset svn:mime-type "application/octet-stream" {} \;
find -iname "*.onetoc2" -exec svn propset svn:mime-type "application/octet-stream" {} \;
find -iname "*.xlsx" -exec svn propset svn:mime-type "application/octet-stream" {} \;
find -iname "*.pdf" -exec svn propset svn:mime-type "application/pdf" {} \;
find -iname "*.ps" -exec svn propset svn:mime-type "application/postscript" {} \;
find -iname "*.gz" -exec svn propset svn:mime-type "application/x-gzip" {} \;
find -iname "*.mdl" -exec svn propset svn:mime-type "application/x-dosexec" {} \;
find -iname "*.zip" -exec svn propset svn:mime-type "application/zip" {} \;



git-svn-id: http://svn.code.sf.net/p/flightgear/fgaddon/trunk/Aircraft/747-8i@997 9eb9bc84-db9d-4e05-893b-26dd41ac94a6

IAHM-COL committed:
SVN updated Revision 1003

IAHM-COL committed:
Adjust Animated Jetway door positions suggested by D-LASER

https://github.com/FGMEMBERS/747-8i/issues/2

IAHM-COL committed:
Livery from the Flightgear Livery database updated

Authors (Thanks to)
	Emmanuel Savage

IAHM-COL committed:
Merge remote-tracking branch 'upstream/master'

Pakistan-1 committed:
Merge remote-tracking branch 'refs/remotes/dogsaysmoo/master'

Pakistan-1 committed:
Merge branch 'master' of https://github.com/FGMEMBERS/747-8i

Pakistan-1 committed:
Merge remote-tracking branch 'refs/remotes/dogsaysmoo/master'

aiannuzzi99 committed:
Air Force One Livery

Air Force One livery from the VIP 747 8i

Commit:
Package variant fixes


git-svn-id: http://svn.code.sf.net/p/flightgear/fgaddon/trunk/Aircraft/747-8i@1472 9eb9bc84-db9d-4e05-893b-26dd41ac94a6

IAHM-COL committed:
Merge remote-tracking branch 'svn/master'

aiannuzzi99 committed:
Blank Green Livery

Blank Green livery added.

aiannuzzi99 committed:
Moved the blank livery to the right place

aiannuzzi99 committed:
Atlas Air Livery

legoboyvdlp committed:
Merge pull request #3 from dogsaysmoo/master

Merge changes Nov 2015 by frank

Commit:
Adapt new location of Aircraft/Generic/marker.xml


git-svn-id: http://svn.code.sf.net/p/flightgear/fgaddon/trunk/Aircraft/747-8i@1622 9eb9bc84-db9d-4e05-893b-26dd41ac94a6

IAHM-COL committed:
Merge remote-tracking branch 'upstream/master'

IAHM-COL committed:
Merge remote-tracking branch 'svn/master'

IAHM-COL committed:
Merge branch 'master' of https://github.com/FGMEMBERS/747-8i

aiannuzzi99 committed:
Emirates Sky Cargo Livery

aiannuzzi99 committed:
British Airways and British Airways One World Liveries

Commit:
747-8i: Package cleanup and minor changes, fixes, and optimizations

git-svn-id: http://svn.code.sf.net/p/flightgear/fgaddon/trunk/Aircraft/747-8i@1702 9eb9bc84-db9d-4e05-893b-26dd41ac94a6

Commit:
747-8: removed unneeded tutorial marker

git-svn-id: http://svn.code.sf.net/p/flightgear/fgaddon/trunk/Aircraft/747-8i@1710 9eb9bc84-db9d-4e05-893b-26dd41ac94a6

IAHM-COL committed:
Merge remote-tracking branch 'upstream/master'

IAHM-COL committed:
Merge SVN commits

IAHM-COL committed:
Merge branch 'master' of https://github.com/FGMEMBERS/747-8i

Pakistan-1 committed:
added merlion cargo livery

DavidBerkut committed:
New Livery: Saudia Cargo

Commit:
747-8: update MCP pick animations

git-svn-id: http://svn.code.sf.net/p/flightgear/fgaddon/trunk/Aircraft/747-8i@2065 9eb9bc84-db9d-4e05-893b-26dd41ac94a6

Commit:
747-8: reverted pick animation update, which was unneeded

git-svn-id: http://svn.code.sf.net/p/flightgear/fgaddon/trunk/Aircraft/747-8i@2078 9eb9bc84-db9d-4e05-893b-26dd41ac94a6

IAHM-COL committed:
Merge remote-tracking branch 'upstream/master'

IAHM-COL committed:
Merge remote-tracking branch 'svn/master'

Commit:
747-8: Add speed-by-pitch mode to AP

git-svn-id: http://svn.code.sf.net/p/flightgear/fgaddon/trunk/Aircraft/747-8i@2133 9eb9bc84-db9d-4e05-893b-26dd41ac94a6

IAHM-COL committed:
Merge remote-tracking branch 'upstream/master'

IAHM-COL committed:
merge svn/master

Pakistan-1 committed:
NEW CX CARGO LIVERY B-LJN

Commit:
747-8: New FDM configurations; new pushback handling

git-svn-id: http://svn.code.sf.net/p/flightgear/fgaddon/trunk/Aircraft/747-8i@2325 9eb9bc84-db9d-4e05-893b-26dd41ac94a6

IAHM-COL committed:
Merge remote-tracking branch 'upstream/master'

Commit:
747-8: Updated exterior lighting to use model combined effect

git-svn-id: http://svn.code.sf.net/p/flightgear/fgaddon/trunk/Aircraft/747-8i@2414 9eb9bc84-db9d-4e05-893b-26dd41ac94a6

Commit:
747-8: Minor tweaks and bug fix for the light mapping

git-svn-id: http://svn.code.sf.net/p/flightgear/fgaddon/trunk/Aircraft/747-8i@2415 9eb9bc84-db9d-4e05-893b-26dd41ac94a6

IAHM-COL committed:
Merge remote-tracking branch 'upstream/master'

Commit:
747-8: nasal modules load earlier than yasim/gross-weight-lbs prop and fail. This fixes that.

git-svn-id: http://svn.code.sf.net/p/flightgear/fgaddon/trunk/Aircraft/747-8i@2527 9eb9bc84-db9d-4e05-893b-26dd41ac94a6

Commit:
747-8: Add metadata tags

git-svn-id: http://svn.code.sf.net/p/flightgear/fgaddon/trunk/Aircraft/747-8i@2639 9eb9bc84-db9d-4e05-893b-26dd41ac94a6

Commit:
747-8: Added de-rated thrust options in CDU and autothrottle

git-svn-id: http://svn.code.sf.net/p/flightgear/fgaddon/trunk/Aircraft/747-8i@2831 9eb9bc84-db9d-4e05-893b-26dd41ac94a6

YV3399 committed:
Merge pull request #4 from dogsaysmoo/master

Merge from orig repo.

IAHM-COL committed:
Merge FGAddon

RenanMsV committed:
Added new wings.png, Varig / KLM livery

New wings.png
Varig Liveries
KLM Orange Pride Livery

ericksonstudio committed:
ignore add

Commit:
747-8i: send sim/multiplay/fallback-model-index.

git-svn-id: http://svn.code.sf.net/p/flightgear/fgaddon/trunk/Aircraft/747-8i@3359 9eb9bc84-db9d-4e05-893b-26dd41ac94a6

Commit:
747-8i: correct sim/model/fallback-model-index.

git-svn-id: http://svn.code.sf.net/p/flightgear/fgaddon/trunk/Aircraft/747-8i@3381 9eb9bc84-db9d-4e05-893b-26dd41ac94a6

Commit:
747-8: Added new and retro Lufthansa liveries

git-svn-id: http://svn.code.sf.net/p/flightgear/fgaddon/trunk/Aircraft/747-8i@3421 9eb9bc84-db9d-4e05-893b-26dd41ac94a6

Commit:
747-8: Update to PAA livery

git-svn-id: http://svn.code.sf.net/p/flightgear/fgaddon/trunk/Aircraft/747-8i@3422 9eb9bc84-db9d-4e05-893b-26dd41ac94a6

IAHM-COL committed:
Merge branch 'fgaddon'
